<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Крутильный маятник</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QToolButton" name="tbSettings" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Настройки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbOscillograph" >
       <property name="text" >
        <string>Осциллограф</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbPendulumView" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Маятник</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="bPause" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Пауза</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="bStart" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Старт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="bStop" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Стоп</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="bExit" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Модуль сдвига и Q¯¹</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" >
       <item rowspan="2" row="0" column="0" >
        <widget class="QwtPlot" name="qwtpMainData" />
       </item>
       <item row="0" column="1" >
        <widget class="QLabel" name="lCorell" >
         <property name="text" >
          <string>0.95</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QSlider" name="vsCorell" >
         <property name="minimum" >
          <number>50</number>
         </property>
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="value" >
          <number>95</number>
         </property>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4" >
      <attribute name="title" >
       <string>Температура</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QwtPlot" name="qwtpTemperature" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QGroupBox" name="groupBox_4" >
       <property name="title" >
        <string>Образец</string>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_20" >
          <property name="text" >
           <string>&lt;b>Длина</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="lPatternLenght" >
          <property name="text" >
           <string>?</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_21" >
          <property name="text" >
           <string>&lt;b>Ширина</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLabel" name="lPatternWidth" >
          <property name="text" >
           <string>?</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_22" >
          <property name="text" >
           <string>&lt;b>Толщина</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLabel" name="lPatternThickness" >
          <property name="text" >
           <string>?</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="label_23" >
          <property name="text" >
           <string>мм</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QLabel" name="label_24" >
          <property name="text" >
           <string>мм</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" >
         <widget class="QLabel" name="label_25" >
          <property name="text" >
           <string>мм</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5" >
       <property name="title" >
        <string>Маятник</string>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_26" >
          <property name="text" >
           <string>&lt;b>Масса m1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="lPendulumM1" >
          <property name="text" >
           <string>?</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="label_27" >
          <property name="text" >
           <string>г</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_28" >
          <property name="text" >
           <string>&lt;b>Масса m2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLabel" name="lPendulumM2" >
          <property name="text" >
           <string>?</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QLabel" name="label_29" >
          <property name="text" >
           <string>г</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_30" >
          <property name="text" >
           <string>&lt;b>Радиус</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLabel" name="lPendulumR" >
          <property name="text" >
           <string>?</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" >
         <widget class="QLabel" name="label_31" >
          <property name="text" >
           <string>мм</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="label_32" >
          <property name="text" >
           <string>&lt;b>Жесткость</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QLabel" name="lPendulumKb" >
          <property name="text" >
           <string>?</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_6" >
       <property name="title" >
        <string>Термокамера</string>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_33" >
          <property name="text" >
           <string>&lt;b>Температура</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="lTemperature" >
          <property name="text" >
           <string>?</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="label_34" >
          <property name="text" >
           <string>°C</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_35" >
          <property name="text" >
           <string>&lt;b>Мощность</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLabel" name="lMetakonPower" >
          <property name="text" >
           <string>?</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" >
         <widget class="QLabel" name="label_36" >
          <property name="text" >
           <string>%</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_37" >
          <property name="text" >
           <string>&lt;b>Скорость</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QLabel" name="label_38" >
          <property name="text" >
           <string>°C/мин</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLabel" name="lHeatingSpeed" >
          <property name="text" >
           <string>?</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Вид</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QCheckBox" name="cbCooling" >
          <property name="text" >
           <string>Охлаждение</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>bExit</sender>
   <signal>clicked()</signal>
   <receiver>bStop</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>644</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel" >
     <x>549</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bExit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>644</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel" >
     <x>349</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
